Agilefreaks/weatherios
# Feature/display weather current location

## Description

Use case
As a user I want to see the weather in my current location so that I know if I need to bring an umbrella

Acceptance criteria
Should display weather info in metric

- summary tittle
- temperature actual
- clouds visibility, humidity

## Related

https://trello.com/c/xtwKeEIa

## Documentation

https://www.apollographql.com/docs/ios/

## Link to demo

![Simulator Screen Shot - iPhone 12 - 2020-12-09 at 14 01 51](https://user-images.githubusercontent.com/15888842/101627486-18414b80-3a27-11eb-922a-ec60a8dbd8f1.png)

## Steps to Test or Reproduce

1. Run app on device
2. Accept location permissions pop-up
3. You should see the main view updated with current location and weather information
